_includes/compact_hub_tags_and_icons.html

```diff
@@ -0,0 +1,8 @@
+<nav class="navbar navbar-expand-lg navbar-light research-menu">
+  {% include hub_model_tags.html category=include.category %}
+  <div class="hub-select-container col-3">
+    {% include hub_icons.html %}
+  </div>
+</nav>
+
+<hr>
```

Comment:
I don't think this includes file is being used anywhere. If that's the case then you can delete it.